.github/workflows/builds.yml

```
@@ -68,33 +75,87 @@ jobs:
           - TYPE: "Debug"
             MSG_LVL: 5
         PLATFORM:
-          - NAME: "CUDA"
-            CONTAINER: "ghcr.io/acts-project/ubuntu1804_cuda:v11"
+          - NAME: "CPU"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:47"
+            CONTAINER_NAME: "Ubuntu 24.04"
+            CXX_STANDARD: "17"
+            OPTIONS:
+            NOTES:
+            RUN_TESTS: true
+          - NAME: "CPU"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:47"
+            CONTAINER_NAME: "Ubuntu 24.04"
+            CXX_STANDARD: "20"
             OPTIONS:
+            NOTES:
+            RUN_TESTS: true
+          - NAME: "CPU"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:47"
+            CONTAINER_NAME: "Ubuntu 24.04"
+            CXX_STANDARD: "23"
+            OPTIONS:
+            NOTES:
+            RUN_TESTS: true
           - NAME: "CUDA"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004_cuda:v32"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2004_cuda:47"
+            CONTAINER_NAME: "Ubuntu 20.04"
+            CXX_STANDARD: "17"
             OPTIONS:
+            NOTES:
+            RUN_TESTS: false
+          - NAME: "CUDA"
+            CONTAINER: "ghcr.io/acts-project/ubuntu1804_cuda:v43"
+            CONTAINER_NAME: "Ubuntu 18.04"
+            CXX_STANDARD: "17"
+            OPTIONS: -DCMAKE_CUDA_STANDARD=14
+            NOTES: CUDA C++14
+            RUN_TESTS: false
           - NAME: "HIP"
-            CONTAINER: "ghcr.io/acts-project/ubuntu1804_rocm:v11"
+            CONTAINER: "ghcr.io/acts-project/ubuntu1804_rocm:v43"
+            CONTAINER_NAME: "Ubuntu 18.04"
+            CXX_STANDARD: "17"
+            OPTIONS: -DCMAKE_HIP_STANDARD=14
+            NOTES:
```

Since I don't want to change the repository's configuration all too often, let's set up the job names consistently to begin with. :thinking:

```suggestion
            NOTES: HIP C++14
```